src/cursor.js

```diff
@@ -198,6 +198,13 @@ class Cursor extends Model {
     return this.editor.scopeDescriptorForBufferPosition(this.getBufferPosition())
   }
 
+  // Public: Retrieves the syntax tree descriptor for the cursor's current position.
+  //
+  // Returns a {ScopeDescriptor}
+  getSyntaxTree () {
```

Comment:
Regarding naming - I think `getSyntaxTree` makes it sound like you are retrieving the entire tree. Since we're returning a scope descriptor, which describes a *path* from the root of the tree to the current node, how about something like:
* `getSyntaxTreePath`
* `getSyntaxTreePathDescriptor`
* `getSyntaxTreeScopeDescriptor`

Thoughts?

Reply:
`getSyntaxTreeScopeDescriptor` sounds the best to me considering the other function is `getScopeDescriptor` and we return a `ScopeDescriptor`. I will use this one!